content/docs/pipelines/sdk/dsl-recursion.md

```diff
@@ -0,0 +1,91 @@
++++
+title = "DSL Recursion"
+description = "Author a recursive function in DSL."
+weight = 7
++++
+
+This page describes how to write recursive functions in DSL.
```

Comment:
Let's give people more of a clue about what we mean by DSL. It's a tech writing pattern to spell out every abbreviation the first time it's used on a page, because we can't assume the reader knows what the abbreviation means.

Suggested wording:

This page describes how to write recursive functions in the domain specific language (DSL) provided by the Kubeflow Pipelines SDK.

Reply:
Good idea. This abbreviation has been used for quite some time and I almost think it is an English word. Fixed.